frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/donut-chart.tsx

```diff
@@ -48,7 +51,10 @@ export const DiscoveryDonutChart: React.FC<DiscoveryDonutChartProps> = ({ state,
             hasDiskType = DISK_TYPES[state.diskType].property === disk.property;
           }
 
-          if (isValidSize && hasDiskType) {
+          const hasDeviceType: boolean = state.deviceType.includes(
+            deviceTypeDropdownItems[disk.type.toUpperCase()],
+          );
```

Comment:
shouldn't this will work?
```suggestion
          const hasDeviceType: boolean = state.deviceType.includes(disk.type.toUpperCase());
```

Reply:
state.deviceType array has values ['Disk', 'Part'], So disk.type.toUpperCase() wont match('DISK' or 'PART')

Reply:
It will if you see this change. 
https://github.com/openshift/console/pull/7924/files#diff-0b015994d7f949840588912fb313db689a1c818fd5a54eab6804e2cfaf8326f1R20

Reply:
The value deviceTypeDropdownItems.DISK or deviceTypeDropdownItems.PART  is  'Disk' / 'Part' only.

export const deviceTypeDropdownItems = Object.freeze({
  DISK: 'Disk',
  PART: 'Part',
});